pages/index.js

```diff
@@ -29,7 +29,7 @@ export default function Home({ isConnected }) {
 
           <Row>
             <Col md={6} lg={4}>
-              <Link href='/todo' passHref>
+              <Link href='/todo' passHref legacyBehavior>
```

Comment:
This legacy behavior will be changed at some point because it will be deprecated and produce errors when latest updates are released from NextJS in the future.